net/connUDP.go

```diff
@@ -24,10 +24,36 @@ type UDPConn struct {
 }
 
 type ControlMessage struct {
+	Dst     net.IP // destination address, receiving only
 	Src     net.IP // source address, specifying only
 	IfIndex int    // interface index, must be 1 <= value when specifying
```

Comment:
Interface-Index 0 is defined as "any interface", so I think the "must be 1<=" part of the comment can be removed.

Reply:
Fixed, thx :)